packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -994,72 +1023,72 @@ export const prepareCosmosOrchestrationAccountKit = (
           return this.state.chainAddress;
         },
         /** @type {HostOf<StakingAccountActions['delegate']>} */
-        delegate(validator, amount) {
+        delegate(validator, amount, opts) {
           return asVow(() => {
             trace('delegate', validator, amount);
-            const { helper } = this.facets;
+            const { helper, holder } = this.facets;
             const { chainAddress } = this.state;
 
             const amountAsCoin = helper.amountToCoin(amount);
 
-            return watch(
-              E(helper.owned()).executeEncodedTx([
+            const result = holder.executeEncodedTx(
```

Comment:
why is it safe to forgo the ownership check? oh, because we're in the holder here.

nit: would have helped review if the `owned()` to `holder` change was a separate refactor